### ISSUE TITLE
More precise error message in case of incorrectly specified DB file

### ISSUE THREAD

**Issue by yurivish** [benbjohnson/litestream#152]
Right now if one configures the database path to point to an invalid DB file, the error is easy to misread as being about the WAL file, rather than the database file. Here's the error I got when I accidentally passed the path of `litestream.yml` as the database file:

```
/Users/yurivish/Dropbox/Projects/associations/litestream.yml: sync error: enable wal: file is not a database
```

This message is technically correct as-is, but perhaps there's a way to rephrase it to emphasize that the user-provided database path does not point to a valid database file.

**Comment by benbjohnson** [comment]
Thanks Yuri! I fixed it up with this PR: https://github.com/benbjohnson/litestream/pull/154
